KD_Lib/original/train.py

```diff
@@ -0,0 +1,57 @@
+import torch.nn.functional as F
+import matplotlib.pyplot as plt
+
+
+def train_teacher(model, train_loader, optimizer, epochs, device):
+
+    model.train()
+    loss_arr = []
+
+    for e in range(epochs):
+        epoch_loss = 0
+        for (data, label) in train_loader:
+            data, label = data.to(device), label.to(device)
+            optimizer.zero_grad()
+            out = model(data)
+            loss = F.cross_entropy(out, label)
+            loss.backward()
+            optimizer.step()
+
+            epoch_loss += loss
+
+        loss_arr.append(epoch_loss)
+        print(f'Epoch {e+1} loss = {epoch_loss}')
+
+    plt.plot(loss_arr)
+    plt.show()
+
+
+def train_student(large_model, distil_model, train_loader, optimizer, loss_fn, epochs=10, temp=20, distil_weight=0.7):
```

Comment:
line length more than 80 chars, please make the args multiline, styling requirements